instrument-starters/opentracing-spring-cloud-jdbc-starter/src/main/java/io/opentracing/contrib/spring/cloud/jdbc/JdbcAspect.java

```diff
@@ -29,6 +30,14 @@
 @Aspect
 public class JdbcAspect {
 
+  private final boolean withActiveSpanOnly;
```

Comment:
withactivespanOnly should be documented. It wasn't clear to me what it means

Reply:
Ok. I documented it https://github.com/opentracing-contrib/java-spring-cloud/pull/177/files#diff-10f92decf61b800616faae597226a6e4R12 and https://github.com/opentracing-contrib/java-spring-cloud/pull/177/files#diff-9cee7f5e8a0570301726d076a1744f86R24 however I am not completely sure that is right or explains it correctly. It's already a field for the `TracingConnection` object. I looked there for some documentation but didn't find much besides https://github.com/opentracing-contrib/java-jdbc/blob/master/README.md#usage

Reply:
I documented it in the `Properties` class instead of here in the `Aspect` class because that was the trend I saw in other parts of this repo